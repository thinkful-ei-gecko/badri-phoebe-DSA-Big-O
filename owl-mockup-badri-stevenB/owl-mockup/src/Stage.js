import React from 'react';


export default function(props){
    console.log(props.abc);
    const theStage = props.abc.filter(val => val.onStage === true).map(val => {
        return <div key={val.id} className='stagePeople'><img src={val.avatar} alt={val.name}/>{val.name}</div>;
    });
    return <section className='stageBox'>{ theStage }</section>;
}