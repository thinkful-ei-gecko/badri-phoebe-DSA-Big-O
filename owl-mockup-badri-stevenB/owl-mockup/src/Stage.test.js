import React from 'react';
import ReactDOM from 'react-dom';
import Stage from './Stage';
import renderer from 'react-test-renderer';

describe('Stage component', () => {
  it('will render', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Stage abc={[1, 2, 3, 4]}/>,div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the UI as expcted', () => {
    const tree = renderer
      .create(<Stage abc={[1, 2, 3, 4]}/>)
      .toJSON();
      expect(tree).toMatchSnapshot();
  });
})