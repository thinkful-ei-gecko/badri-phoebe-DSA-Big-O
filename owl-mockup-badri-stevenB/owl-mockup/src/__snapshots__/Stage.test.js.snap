// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage component renders the UI as expcted 1`] = `
<section
  className="stageBox"
/>
`;
